alerting/src/main/kotlin/org/opensearch/alerting/MonitorRunner.kt

```diff
@@ -184,4 +186,9 @@ abstract class MonitorRunner {
     protected fun createMonitorMetadata(monitorId: String): MonitorMetadata {
         return MonitorMetadata("$monitorId-metadata", monitorId, emptyList(), emptyMap())
     }
+
+    fun shouldProcessTrigger(trigger: Trigger, maxActionsPerTrigger: Int): Boolean {
```

Comment:
It should be should process action, we do not want the trigger to stop executing. We only want to stop the extra actions from being executed. So this means that calling this in the different monitors should only limit the actions and not triggers from executing.